### DISCUSSION TITLE
能否支持代理设置，经常卡住

### DISCUSSION THREAD

**Discussion by zhanxh** [ayamir/nvimdots#1350] [Q&A] [answered]
能否支持代理设置，经常卡住

**Comment by ayamir** [comment, marked as answer]
You can set `settings["use_ssh"] = false` in `user/settings.lua` to bypass gfw limitation if you don't have a proxy server to relay github.
If you have a proxy setting, you should enable it in terminal like `export http_proxy=http://127.0.0.1:7890` and set git config to use proxy server: https://gist.github.com/evantoli/f8c23a37eb3558ab8765
